components/image/image.css

```diff
@@ -0,0 +1,11 @@
+/* || RESPONSIVE IMAGE */
+.img-responsive {
+  max-width: 100%;
+  height: auto;
```

Comment:
instead of height auto use object-fit : cover/contain

Reply:
Ya will do it. Thanks!